packages/legend-extension-external-language-morphir/src/MorphirGenerationPlugin.tsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import packageJson from '../package.json';
+
+import type {
+  DSL_StudioPlugin_Extension,
+  GenerationFile,
+  FileGenerationState,
+  StudioPluginManager,
+  ExtraFileGenerationtButton,
+} from '@finos/legend-studio';
+
+import { StudioPlugin } from '@finos/legend-studio';
+import { NetworkClient } from '@finos/legend-shared';
+
+export class MorphirGenerationPlugin
+  extends StudioPlugin
+  implements DSL_StudioPlugin_Extension
+{
+  networkClient: NetworkClient;
+
+  constructor() {
+    super(packageJson.extensions.studioPlugin, packageJson.version);
+    this.networkClient = new NetworkClient();
+  }
+
+  install(pluginManager: StudioPluginManager): void {
+    pluginManager.registerStudioPlugin(this);
+  }
+
+  visualizeMorphir =
+    (fileNode: GenerationFile): (() => void) =>
+    (): void => {
+      this.networkClient.post(
+        `http://0.0.0.0:9901/insight`,
```

Comment:
I thought we said we would allow configuring this via plugin config?

Reply:
Added configure for Morphir plugin which sets url fields.